src/app/modules/main/module.nim

```diff
@@ -118,6 +118,8 @@ type
     pendingSpectateRequest: SpectateRequest
     statusDeepLinkToActivate: string
 
+{.push warning[Deprecated]: off.}
```

Comment:
Would it be possible to disable deprecated warnings on compiler level rather than adding this pragma into multiple module files?